vmdb/app/assets/javascripts/miq_dhtmlxtoolbar.js

```diff
@@ -157,7 +157,7 @@ function miqToolbarOnClick(id) {
     } else {
       // An ajax url was defined
       tb_url = "/" + miq_controller + "/" + button.url;
-      if (button.url.indexOf("x_history") != 0) {
+      if (button.url.indexOf("x_history")) {
```

Comment:
I think this one warrants a `!== 0` just to be clear we're doing a `! startsWith`.